src/ec_log.c

```diff
@@ -37,8 +37,11 @@
 
 /* globals */
 
-static struct log_fd fdp;
-static struct log_fd fdi;
+/* zero is formally a valid value for an opened file descriptor
```

Comment:
Yep, as it is well known, it is preconnected to stdin. But a user can close this descriptor at the program launching, e.g. from the bash command line
$ ettercap 0<&-
And posix standard  says that: "The open() function shall return a file descriptor for the named file that is the lowest file descriptor not currently open for that process."
I don't know how it would be in real life -- whether open() will return 0 in the case if it is a lowest free file descriptor or not.